workshops/templates/workshops/instructors.html

```diff
@@ -10,7 +10,7 @@
 {% endblock %}
 
 {% block content %}
-    {% crispy form form_helper %}
+    {% crispy form %}
 {% if persons == None %}
```

Comment:
Should I squash the four commits into one since it is an atomic update. Is that right?